src/codemagic/tools/app_store_connect.py

```diff
@@ -148,6 +154,72 @@ def _delete_resource(self, resource_manager, resource_id: ResourceId, ignore_not
             else:
                 raise AppStoreConnectError(str(api_error))
 
+    @cli.action('list-testflight-builds',
+                BuildArgument.APPLICATION_ID_OPTIONAL,
+                BuildArgument.EXPIRED,
+                BuildArgument.NOT_EXPIRED,
+                BuildArgument.BUILD_ID_OPTIONAL,
+                BuildArgument.PRE_RELEASE_VERSION,
+                BuildArgument.PROCESSING_STATE,
+                BuildArgument.BUILD_NUMBER,
+                BuildArgument.ORDERING,
+                BuildArgument.REVERSE)
+    def list_testflight_builds(self,
+                              application_id: Optional[Union[str, ResourceId]] = None,
+                              expired: Optional[bool] = None,
+                              not_expired: Optional[bool] = None,
+                              build_id: Optional[Union[str, ResourceId]] = None,
+                              pre_release_version: Optional[str] = None,
+                              processing_state: Optional[BuildProcessingState] = None,
+                              build_number: Optional[str] = None,
+                              ordering: Optional[BuildOrdering] = None,
+                              reverse: Optional[bool] = False,
+                              should_print: bool = True) -> List[Build]:
+        """
+        List Testflight builds
+        """
+        try:
+            expired_value = get_binary_arguments_value(expired, not_expired)
+        except:
```

Comment:
Plain `except:` clause is considered to be anti-pattern and instead you should catch as specific exception as possible.

Reply:
fix [here](https://github.com/codemagic-ci-cd/cli-tools/pull/67/commits/71f2ca10806aa46be85d0feca84182780939f3d7#diff-c655fbe88a18ddd4d367405c06673842a69855a44563b37bd3b06625bf712419R178)